runtime/lua/vim/treesitter/query.lua

```diff
@@ -74,7 +74,7 @@ end
 ---@return string[] query_files List of files to load for given query and language
 function M.get_files(lang, query_name, is_included)
   local query_path = string.format('queries/%s/%s.scm', lang, query_name)
-  local lang_files = dedupe_files(a.nvim_get_runtime_file(query_path, true))
+  local lang_files = dedupe_files(api.nvim_get_runtime_file(query_path, true))
```

Comment:
```suggestion
  local lang_files = dedupe_files(vim.api.nvim_get_runtime_file(query_path, true))
```
Could inline this, as it's the only use in that file?

Reply:
There were a couple `vim.api.*` that could use this.

Reply:
Maybe change (at least one of) those then to make it worth it? ;)

Reply:
I did

Reply:
(sorry, only saw the reply on mail, not the actual changes)